components/api_request_helper/api_request_helper.cc

```diff
@@ -8,15 +8,35 @@
 #include <utility>
 
 #include "net/base/load_flags.h"
+#include "services/data_decoder/public/cpp/json_sanitizer.h"
 #include "services/network/public/cpp/resource_request.h"
 #include "services/network/public/cpp/shared_url_loader_factory.h"
 #include "services/network/public/cpp/simple_url_loader.h"
 #include "services/network/public/mojom/url_response_head.mojom.h"
 
 namespace api_request_helper {
 
+namespace {
+
+void OnSanitize(const int http_code,
+                const base::flat_map<std::string, std::string>& headers,
+                APIRequestHelper::ResultCallback result_callback,
+                data_decoder::JsonSanitizer::Result result) {
+  std::string response_body;
+  if (result.value.has_value())
+    response_body = result.value.value();
```

Comment:
Move this after result.error check block, as these are not needed if we error out.

Reply:
done